backend/src/main/java/quarano/actions/DiaryEventListener.java

```diff
@@ -88,7 +104,15 @@ private boolean determineWhetherEntryMostRecent(TrackedPersonIdentifier personId
 				.isEmpty();
 	}
 
-	void handleDiaryEntryForCharacteristicSymptoms(DiaryEntry entry) {
+	/**
+	 * manage the list of characteristic symptoms - the resulting action depends
+	 * @param entry
+	 * @param isContactCase true if person is contact of an indexed case, false otherwise
+	 */
+	void handleDiaryEntryForCharacteristicSymptoms(DiaryEntry entry, boolean isContactCase) {
+		if (!isContactCase) {
```

Comment:
Seems to validate my suspicion above. If the flag handed in immediately aborts the method call, why call the method in the first place?

Reply:
yes, greatly simplifies the sitution! Now, if we could say for sure that this fragment is also contact case specific - both method calls could be parked in a single if:

		items.findUnresolvedByDescriptionCode(person, DescriptionCode.INCREASED_TEMPERATURE)
				.resolveAutomatically(items::save);